hack/generate/csv.sh

```diff
@@ -14,7 +14,7 @@ export CURRENT_VERSION_IMAGES=${CURRENT_VERSION_IMAGES:-"nightly"}
 
 client_version="$(metadata.get dependencies.cli)"
 kn_event="${registry_host}/knative/release-${client_version%.*}:client-plugin-event"
-rbac_proxy="registry.ci.openshift.org/origin/$(metadata.get 'requirements.ocpVersion.min'):kube-rbac-proxy"
+rbac_proxy="registry.ci.openshift.org/origin/$(metadata.get 'requirements.ocpVersion.max'):kube-rbac-proxy"
```

Comment:
We can revert back to .min value once http/2 fix is cherry-picked to 4.11 midstream.